setup.py

```diff
@@ -86,14 +92,19 @@ def find_javac(possible_homes):
     return name  # Fall back to "hope it's on the path"
 
 
-def compile_native_invocation_handler(*possible_homes):
+def compile_native_invocation_handler(*args):
```

Comment:
Instead of renaming like this, what about making `possible_homes` a constant and not renaming the old argument to name saying nothing?